### PR TITLE
WL-4783 Ensure instruction only displayed if required.

### PR DESCRIPTION
**Pull request by RebeccaMiller-Which**
Also change text of second instruction.

### CODE REVIEW

File: site-manage/site-manage-tool/tool/src/webapp/vm/sitesetup/chef_site-modifyENW.vm [hunk 1]
```diff
@@ -55,7 +58,7 @@
 		<p class="instruction">
 			$tlang.getString("multipleTools.custom1")
 		</p>
-			#foreach($toolId in $!toolRegistrationSelectedList) 
+			#foreach($toolId in $!toolRegistrationSelectedList)
```
Comment (nicholaswilson100) [inline site-manage/site-manage-tool/tool/src/webapp/vm/sitesetup/chef_site-modifyENW.vm]: Whitespace

File: site-manage/site-manage-tool/tool/src/webapp/vm/sitesetup/chef_site-modifyENW.vm [hunk 2]
```diff
@@ -31,13 +31,16 @@
...
 	#if (!$!ltiTools.isEmpty())
-		<p class="instruction">
-			$tlang.getString("ltiTools.custom.instruction")
-		</p>
 		#foreach($ltiTool in $ltiTools.entrySet())
 			#set($ltiToolId=$ltiTool.getKey())
 			#if (!$!oldLtiTools.containsKey($ltiToolId))
-				## show the input only if the tool is not enabled in the site yet
+				## show the input only if the tool is not enabled in the site yet and display the heading only once
+				#if (!$!instruction)
```
Comment (nicholaswilson100) [inline site-manage/site-manage-tool/tool/src/webapp/vm/sitesetup/chef_site-modifyENW.vm:38]: What does !$! do - did you find out in the end? Test for empty list?

**Comment by buckett** [comment]
Comment in jira about this one.

**Comment by buckett** [comment]
@RebeccaOxfordUni Ping.

File: site-manage/site-manage-tool/tool/src/webapp/vm/sitesetup/chef_site-modifyENW.vm (hunk 2 above)
Comment (RebeccaMiller-Which) [reply site-manage/site-manage-tool/tool/src/webapp/vm/sitesetup/chef_site-modifyENW.vm:38]: I think the $! means no exception if instruction is null and the first ! is a logical not.

**Comment by RebeccaMiller-Which** [comment]
Rogue space added back in.

**Review by buckett — APPROVED** [review]

**Pull request merged**